Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -121,6 +129,14 @@ namespace winrt::BabylonReactNative::implementation {
         const auto position = point.Position();
         const uint32_t x = position.X < 0 ? 0 : static_cast<uint32_t>(position.X);
         const uint32_t y = position.Y < 0 ? 0 : static_cast<uint32_t>(position.Y);
+        const auto pointerId = point.PointerId();
+
+        _pressedPointers.erase(pointerId);
+        if (_pressedPointers.size() == 0 &&
+            _inputSource.HasCapture())
+        {
+            _inputSource.ReleasePointerCapture();
+        }

```

Comment:
Is it possible to have two different pointer types active at the same time (e.g. mouse and touch)?

Reply:
I believe so. Input source pointer events seem capable of exposing pen/touch/mouse. I don't personally have a way to test this currently.